cqrs-codegen/src/event/common.rs

```diff
@@ -0,0 +1,109 @@
+//! Shared implementation details for [`cqrs::Event`] proc-macro-derive family
```

Comment:
Just put this in `mod.rs` and it will be shared for sub-modules without requiring of `pub` level visibility. No need in separate `common` module.